docs/ide-usage.md

```diff
@@ -32,7 +32,7 @@ The following plugins are needed:
 
      ![setupguide-3.png](images/setupguide-3.png)
 
-   Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
+     Note that none of the App Engine SDK or JDK to be used are required to be the `default`.
```

Comment:
Revert this. It may look trivial, but in markdown syntax, this changes the indentation level.